lib/services/role.go

```diff
@@ -349,6 +349,7 @@ func validateRoleExpressions(r types.Role) error {
 			{"db_labels", types.KindDatabase},
 			{"db_service_labels", types.KindDatabaseService},
 			{"windows_desktop_labels", types.KindWindowsDesktop},
+			{"windows_desktop_labels", types.KindDynamicWindowsDesktop},
```

Comment:
since `GetLabelMatchers` returns the same expression I think there's no need for this, it's just validating the same expression twice

Reply:
Should we leave it here for consistency sake?